Tools/WinMLRunner/WinMLRunner.sln

```diff
@@ -12,31 +12,37 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "WinMLRunnerTest", "..\..\Te
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
+		Release|Any CPU = Release|Any CPU
 		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{81EA9CC6-8A26-4583-B1A4-84740EF815C8}.Debug|Any CPU.ActiveCfg = Debug|Win32
```

Comment:
When we specify Any CPU Debug, it'll build to Win32 Debug

Reply:
Also, Debug|Any CPU.Build.0 is missing. If we click "Build solution" with the configuration: Any Debug | Any CPU, then WinMLRunner won't build.